relationship.py

```diff
@@ -0,0 +1,101 @@
+#Author Ammanuel Amare
+#9/5/22 :)
+from UMLClass import *
+# This list is used to store the relationships. 
+listofrelationships=[]
+class relationship:
+    # This is the constructor, you will need 3 paramaters to initialize the class
+    # Every relationship has a source ,destination, and a type
+    def __init__(self,type,source,destination):
+        self.type = type
+        self.destination = destination
+        self.source = source
+
+#@para source is thesource class for relationship 
+#@para Destination is the destination class for relationship
+#@para tpe is the type of relationship defined between the classes
+# returns a string with needed information, can be a boolean value in the future. 
+#this is how you will add relationship to class
+def Add_relationship(source: str, destination: str, type: str):
+    
+    status =""
+# To add a class to a relationship it must exist, so we will search for the class,
+# if it does not exist we will fail with a error message. 
+    sourceclass = findClass(source)
+    destinationclass = findClass(destination)
+    #both lists will be needed for testing.
+    # this will take the second item of the tuple and convert it into a list 
+    listofdestinationclasses =(list(list(zip(*tuple_list))[1]))
+    
+    # this will take the first item of the tuple and convert it into a list 
+    listofsourceclasses =(list(list(zip(*tuple_list))[0]))
+
+    if sourceclass is not None and destinationclass is not None:
+        # Create tuple to later be appened to list of relationships.
+        newrelationship = (sourceclass, destinationclass,type)
```

Comment:
I know we discussed storing the relationships as a tuple, but after some thinking, wouldn't it be better to create a Relationship object and store that instead? Otherwise the Relationship class is never used.

Reply:
Sure, when’s the relationship class going to be finished?

Reply:
Other than the object storage vs tuple Do you have any other design changes you want to make to the relationship class?